src/main/kotlin/uk/gov/justice/digital/hmpps/hmppshdcapi/licences/HdcLicence.kt

```diff
@@ -161,10 +161,19 @@ data class AddressAndPhone(
 data class Rejection(
   val address: CurfewAddress?,
   val addressReview: AddressReviewWrapper? = null,
-  val riskManagement: RiskManagement? = null,
+  val riskManagement: RejectedRiskManagement? = null,
   val withdrawalReason: String? = null,
 )
 
+@JsonInclude(NON_NULL)
+data class RejectedRiskManagement(
+  val proposedAddressSuitable: String? = null,
+  val unsuitableReason: String? = null,
+  val manageInTheCommunity: String? = null,
+  val manageInTheCommunityNotPossibleReason: String? = null,
+  val hasConsideredChecks: String? = null,
+)
```

Comment:
we copy over the risk management section when an address is rejected but not all of it - only these fields